package.json

```diff
@@ -40,8 +46,12 @@
   "scripts": {
     "pretest": "npm run build-standalone",
     "test": "jest",
+    "prepublishOnly": "npm run build",
     "prettier": "prettier --plugin=. --parser=php",
-    "build-standalone": "rollup -c build/rollup.config.js"
+    "build": "npm run build-standalone && npm run build-standalone-esm && npm run build-standalone-cjs",
+    "build-standalone": "rollup -c build/rollup-umd.config.js",
+    "build-standalone-esm": "rollup -c build/rollup-esm.config.js",
+    "build-standalone-cjs": "rollup -c build/rollup-cjs.config.js"
```

Comment:
Hm, we really need this types of `standalone`? prettier have only one

/cc @suchipi

Reply:
That's because Prettier does not have a dependency on Prettier. 😁

The default `standalone.js` build relies on the presence of a global `prettier` which is not available in a bundler context. The other two builds `require`/`import` Prettier from `prettier/standalone`.

However, there may be a way to combine both approaches (use `require()` when in bundler context, use a global variable in browsers) but I'm not sure if Rollup supports generating anything like that.

If we'd find a way to generate such a unified UMD bundle, we could get rid of the `esm` and `cjs` builds.

Reply:
I've omitted the minification step and took a closer look at the generated UMD bundle. It seems that it actually *does* exactly that — it currently just `require()`s `prettier` instead of `prettier/standalone`, I'll take a look if we can fix that.

Reply:
:+1:

Reply:
Wow, I've actually got it to work with a single build. Didn't expect that.

Thanks for staying sceptical!

I'll attach another commit and remove the unnecessary builds.

Reply:
Alright, we're back to one build. 👍